go.mod

```diff
@@ -4,14 +4,13 @@ go 1.16
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0 // indirect
-	github.com/golang/protobuf v1.4.0 // indirect
```

Comment:
I'm not sure why this was removed or it shouldn't have been there from the start...
Any ideas @hickeyma ?